sample_apps/run_example.sh

```diff
@@ -1108,7 +1108,7 @@ function get_measurement_of_trusted_simple_app_under_islet() {
     # get_measurement_of_app_by_name "islet_example_app.exe"
 
     run_cmd
-    local mrenclave="6190EB90B293886C172EC644DAFB7E33EE2CEA6541ABE15300D96380DF525BF9"
+    local mrenclave="580bd77074f789f34841ea9920579ff29a59b9452b606f73811132b31c689da9"
```

Comment:
Thanks for figuring this out ... This hash probably was one of the core causes for the failure.

Qs. How often do you think this kind of hash will change?

I hard-coded it in this script (way long ago) to just get something going.

If we do need to make this more robust, to avoid a hard-coded string, we will need to figure out someway to automatically 'generate' this hash-ID and `grep` for it here, from some other source.

If you think all that hoopla is needed, then feel free to log a different issue, recording the potential problem, and describe a potential solution. One of us (two) could look into improving the situation in future.

On the other hand, if you think this kind of hard-coding is acceptable and it won't change too much, then we can live with it.